### PR TITLE
[FIX] web: property fields should take all width

### PR DESCRIPTION
**Pull request by adr-odoo**
Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/web/static/src/views/fields/properties/properties_field.xml [hunk 1]
```diff
@@ -1,60 +1,62 @@
...
+            <t t-foreach="groupedPropertiesList" t-as="PropertiesListGroup" t-key="PropertiesListGroup_index">
+                <div class="o_inner_group grid col-lg-6">
+                    <div
+                        t-foreach="PropertiesListGroup"
+                        t-as="propertyConfiguration"
+                        t-key="propertyConfiguration.name"
+                        class="o_property_field d-flex flex-column d-sm-contents"
+                        t-att-property-name="propertyConfiguration.name">
+                        <t t-set="domId" t-value="generateUniqueDomID()"/>
+                        <div class="text-break">
```
Comment (pparidans) [inline addons/web/static/src/views/fields/properties/properties_field.xml:14]: nécessaire que ce soit sur un parent? Ne peut pas être sur le `<label>` directement?

File: addons/web/static/src/views/fields/properties/properties_field.xml [hunk 2]
```diff
@@ -1,60 +1,62 @@
...
-                            <i t-else="" class="o_field_property_empty_label">
-                                New Property
-                            </i>
-                            <i
-                                t-if="state.canChangeDefinition &amp;&amp; !props.readonly"
-                                class="o_field_property_open_popover fa fa-pencil ms-2"
-                                t-on-click="(event) => this.onPropertyEdit(event, propertyConfiguration.name)"/>
-                        </label>
+        <div t-ref="properties" class="row align-items-start">
+            <t t-foreach="groupedPropertiesList" t-as="PropertiesListGroup" t-key="PropertiesListGroup_index">
```
Comment (pparidans) [inline addons/web/static/src/views/fields/properties/properties_field.xml:5]: `PropertiesListGroup_index` :thinking:

Comment (pparidans) [reply addons/web/static/src/views/fields/properties/properties_field.xml:5]: `PropertiesListGroup` shouldn't start with uppercase

Comment (adr-odoo) [reply addons/web/static/src/views/fields/properties/properties_field.xml]: done

File: addons/web/static/src/views/fields/properties/properties_field.xml (hunk 1 above)
Comment (adr-odoo) [reply addons/web/static/src/views/fields/properties/properties_field.xml:14]: Si mais c'était pour respecter le template de la vue form.

**Comment by adr-odoo** [comment]
@robodoo delegate=rfr-odoo

**Review by rfr-odoo — APPROVED** [review]
@robodoo r+

**Pull request closed without merging**